packages/flutter/lib/src/widgets/notification_listener.dart

```diff
@@ -50,37 +50,14 @@ abstract class Notification {
   /// const constructors so that they can be used in const expressions.
   const Notification();
 
-  /// Applied to each ancestor of the [dispatch] target.
-  ///
-  /// The [Notification] class implementation of this method dispatches the
-  /// given [Notification] to each ancestor [NotificationListener] widget.
-  ///
-  /// Subclasses can override this to apply additional filtering or to update
-  /// the notification as it is bubbled (for example, increasing a `depth` field
-  /// for each ancestor of a particular type).
-  @protected
-  @mustCallSuper
-  bool visitAncestor(Element element) {
```

Comment:
Even though nothing broke in customer tests or g3, do you think we should leave this and mark it deprecated? I could also leave the old dispatch path and instead add a new method to do a fast dispatch

Reply:
leaving this and the old dispatch in as `deprecated` sounds reasonable to me. I just wonder if we can find a reasonable new name for the new dispatch. `fastDispatch` sounds strange after we remove the regular `dispatch` - and what if we find an even faster dispatch in the future?

Reply:
maybe I could _just_ leave the BuildContext dispatch and deprecate the dispatch on Notification?

Reply:
that is, BuildContext would have the new dispatch and Notification would keep the old

Reply:
That sounds reasonable. We'd just have to check all docs pointing to it on the Notification and update them to point to BuildContext instead.